compiler/match_/inference/type_seeder.rs

```diff
@@ -720,7 +720,7 @@ impl BinaryConstraint for Has<Variable> {
             } // It can't be another type => Do nothing and let type-inference clean it up
         };
         attribute
-            .get_owns(seeder.snapshot, seeder.type_manager)?
```

Comment:
We used to have 
`get_owns` and `get_owns_declared` for `AttributeType`. However, `_declared` is confusing here, because you don't fully understand what are the additional `ObjectType`s returned in `get_owns`. So it's just called `get_owner_types` now (all owner types for this attribute type), and `get_owns_declared` is now `get_owns` of this `attribute_type` (not `ObjectType`s, but `Owns`!)